### PR TITLE
adds support for list lpos command

### PR DESCRIPTION
**Pull request by kenus34**
This commit adds support for the `lpos` command added in Redis 6.0.6

### CODE REVIEW

**Comment by kenus34** [comment]
@sazzad16 Thanks a lot. I have made the changes and pushed the same.

File: src/main/java/redis/clients/jedis/commands/Commands.java [hunk 1]
```diff
@@ -17,6 +17,8 @@
 import redis.clients.jedis.params.SetParams;
 import redis.clients.jedis.params.ZAddParams;
 import redis.clients.jedis.params.ZIncrByParams;
+import redis.clients.jedis.params.LPosParams;
+import redis.clients.jedis.util.SafeEncoder;
```
Comment (sazzad16) [inline src/main/java/redis/clients/jedis/commands/Commands.java:21]: Not needed.
```suggestion
```

File: src/test/java/redis/clients/jedis/tests/commands/ListCommandsTest.java [hunk 2]
```diff
@@ -328,6 +329,86 @@ public void lpop() {
 
   }
 
+  @Test
+  public void lpos() {
```
Comment (sazzad16) [inline src/test/java/redis/clients/jedis/tests/commands/ListCommandsTest.java:333]: Test empty LPosParams.

Also, move this test method after all List related tests.

**Review by sazzad16 — CHANGES_REQUESTED** [review]
Almost done!

File: src/main/java/redis/clients/jedis/commands/Commands.java (hunk 1 above)
Comment (kenus34) [reply src/main/java/redis/clients/jedis/commands/Commands.java]: Removed stray import

File: src/test/java/redis/clients/jedis/tests/commands/ListCommandsTest.java (hunk 2 above)
Comment (kenus34) [reply src/test/java/redis/clients/jedis/tests/commands/ListCommandsTest.java]: Refactored test towards the done and added a assertion with an empty paras list.

**Comment by kenus34** [comment]
Thanks for the suggestion. I have resolved the comments.

**Review by sazzad16 — APPROVED** [review]

**Review by gkorland — APPROVED** [review]

**Pull request merged**